#!/usr/bin/env python

PACKAGE = "control"

from dynamic_reconfigure.parameter_generator_catkin import *


gen = ParameterGenerator()

gen.add("manual_override_active", bool_t, 0, "Activate Manual Override", False)
gen.add("manual_steer", double_t, 0, "Steering input sent to carla.", 0.0, -1, 1)
gen.add("manual_throttle", double_t, 0, "Throttle input sent to carla. (Negative is reverse)", 0.0, -1.0, 10.0)
gen.add("stanley_off", bool_t, 0, "Deactivate stanley. Only use Pure pursuit", True)


exit(gen.generate(PACKAGE, "control", "Controller"))
